Sources/SwiftParser/StringLiterals.swift

```diff
@@ -598,3 +598,105 @@ public extension RawTriviaPiece {
     }
   }
 }
+
+// MARK: - StringLiteralExprSyntax/contentValue
+
+extension StringLiteralExprSyntax {
```

Comment:
Since this really is functionality that’s separate from the parser, I think it should live in a separate file. That way it’s nice and isolated.

Reply:
Agreed.